apps/nextjs/src/components/absence.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+
+interface AbsenceProps {
+  data: {
+    userId: number;
+    absenceDate: Date;
+  };
```

Comment:
probably better to directly expose these fields instead of just putting them in `data`